rootfs/bin/doctor

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eo pipefail
+uid=`curl --silent --show-error  -X POST -d "skip" localhost:8080/doctor`
```

Comment:
```console
$ shellcheck doctor

In doctor line 3:
uid=`curl --silent --show-error  -X POST -d "skip" localhost:8080/doctor`
    ^-- SC2006: Use $(..) instead of legacy `..`.
```
Just a suggestion. Maybe we could (later) enable `shellcheck` in `make test-style`?